sumtool/storage.py

```diff
@@ -74,9 +105,7 @@ def store_model_summaries(
             stored_summaries = json.load(f)
 
     for document_id, summary in generated_summaries.items():
-        if model_config_hash not in stored_summaries:
-            stored_summaries[model_config_hash] = {}
```

Comment:
why this change?

Reply:
Because we no longer index summaries by the `model_config_hash`. They are indexed directly by the summary id.